Adding shutDown method

ml-rdv opened the pull request.

ml-rdv committed:
Adding shutDown method

Adgam commented on src/InputOutputStream/ConsoleTextFileEditor/ConsoleTextFileEditor.java:
```diff
@@ -53,7 +54,7 @@ public void start() {
+        while (isActivated) {
             command = in.nextLine();
             clearConsole();
             if (command.equals("finish")) {
```
команду "finish" теперь можно перенести в обработчик команд и там вызывать метод shutdown

ml-rdv replied:
@Adgam fixed

ml-rdv committed:
Move finish into roadCommand method

ml-rdv committed:
Add DeleteResponse,
Check if args.length == 0 in Main
Rename roadCommand to handleCommand
Invert getTextFileContents method
Remove result
Correct tests

Adgam commented on src/InputOutputStream/ConsoleTextFileEditor/ConsoleTextFileEditor.java:
```diff
@@ -222,25 +222,30 @@ public void printInfo(String input) {
-                && fileSystemResponse.getMessageError().equals("Directory is not empty.")) {
+        FileSystemResponse<DeleteResponse> fileSystemResponse = manager.delete(path);
+
+        if (fileSystemResponse.getBody().isDeleted()) {
```
хочется внести немного ясности
```java
        DeleteResponse deleteResponse = fileSystemResponse.getBody();
        
        if (deleteResponse.isDeleted()) {
            return fileSystemResponse.getMessageInfo();
        }
        if (deleteResponse.notExist()) {
            System.out.println("Directory or file does not exist.");
            return fileSystemResponse.getMessageInfo();
        } 
        if (deleteResponse.isNotEmpty()) {
```
если `else if` ещё можно оставить, то выделить deleteResponse хорошо, т.к. по `fileSystemResponse.getBody()` сложно понять, что конкретно лежит в body

ml-rdv replied:
@Adgam fixed

Adgam commented on src/InputOutputStream/ConsoleTextFileEditor/FileSystemManagement.java:
```diff
@@ -155,56 +144,48 @@ private void deleteDirectory(File directory) {
             directory.delete();
-            result = new FileSystemResponse<>(String.format("Directory or file %s has been successfully deleted.\n", directory.getName()));
-            return result;
+            return new FileSystemResponse<>(new DeleteResponse(true),
```
вспоминай про читаемость

либо
```java
            new FileSystemResponse<>(
                    new DeleteResponse(false, false, true),
                    String.format("Directory or file %s has been not deleted.\n", directory.getName())
            );
```

а лучше
```java
            DeleteResponse deleteResponse = new DeleteResponse(false, false, true);
            String message = String.format("Directory or file %s has been not deleted.\n", directory.getName());
            new FileSystemResponse<>(deleteResponse, message);
```

ml-rdv replied:
@Adgam fixed

Adgam commented on src/InputOutputStream/ConsoleTextFileEditor/FileSystemResponse.java:
```diff
@@ -2,30 +2,30 @@
 public class FileSystemResponse<T> {
     private T body;
-    private String messageError;
+    private String messageInfo;
```
после переименования теперь метод createErrorResponse выглядит как то не согласовано

ml-rdv replied:
@Adgam fixed

ml-rdv committed:
Corrections
